src/bun_cpp_utils.cpp

```diff
@@ -5,23 +5,30 @@
 
 #include <cxxabi.h>
 
+template<typename T>
+struct deleter
+{
+	void operator() (T data) {
+		free(data);
+	}
+};
+
 bool
 bun_unwind_demangle(char *dest, size_t dest_size, const char *src)
 {
-	char *result;
-
-	result = abi::__cxa_demangle(src, nullptr, nullptr, nullptr);
-	if (result) {
-		size_t len = strlen(result);
+	// TODO: T14745
+	std::unique_ptr<char, deleter<char*>> result(abi::__cxa_demangle(src, nullptr, nullptr, nullptr), deleter<char*>());
+	char* result_cstr = result.get();
```

Comment:
Don't do that: stashing the value of `unique_ptr::get` in variables makes it easier to have dangling pointers. We should instead call `get` everywhere we need a raw pointer.